fossunited/chapters/doctype/foss_chapter/foss_chapter.json

```diff
@@ -206,6 +207,12 @@
    "fieldtype": "Data",
    "label": "X",
    "options": "URL"
+  },
+  {
+   "default": "/files/City_Illustration.svg",
```

Comment:
I don't think this is gonna carry over to the website. It will break

Reply:
It wont but we cam add a similar file in the cloud desk too right - that will make it a default, until we edit the image for different attachement, of an image representing a city.